modules/tinymce/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Inserting newlines inside an editable element inside a noneditable root would sometimes try to split the editable element. #TINY-9461
 - Creating a list in a table cell when the caret is in front of an anchor element would not properly include the anchor in the list. #TINY-6853
 - Formatting could be applied or removed on noneditable list items inside a noneditable root. #TINY-9563
+- Removing annotation is possible immediately after creation without any selection changes. #TINY-9399
```

Comment:
```suggestion
- Annotation would not be removed if the conversation is immediately deleted after the creation.. #TINY-9399
```

Reply:
Not sure we should mention `conversation` in the changelog of tinymce since that is comment specific.

Maybe something like this:
```suggestion
- Annotation would not be removed if the annotation was immediately deleted after being created. #TINY-9399
```

Reply:
Thanks! Updated with the suggestions.